streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java

```diff
@@ -21,19 +21,32 @@
 
 /** @since 1.2.3 */
 public enum ClusterState implements Serializable {
+
   /** The cluster was just created but not started */
   CREATED(0),
+
   /** cluster started */
   RUNNING(1),
+
   /** cluster stopped */
   CANCELED(2),
+
   /** cluster lost */
   LOST(3),
+
   /** cluster unknown */
   UNKNOWN(4),
+
+  /** cluster starting */
   STARTING(5),
+
+  /** cluster cancelling */
   CANCELLING(6),
+
+  /** cluster failed */
   FAILED(7),
+
+  /** cluster killed */
   KILLED(8);
 
   private final Integer value;
```

Comment:
How about changing it to `private final Integer state;`

Reply:
I think it's ok